pkg/dataplane/const.go

```diff
@@ -71,4 +71,7 @@ const (
 
 	//HostnameLabel label for marking secrets to be watched for changes
 	HostnameLabel = "hostname"
+
+	//AllNodeSets
```

Comment:
Please give this constant further annotation, just to make clear what is it for.

Reply:
OK, this constant is mostly internal usage, added a verbose comment.